nix/options.nix

```
@@ -39,6 +39,15 @@ in
       '';
     };
 
+    deployment.provisionSSHKey = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        This option specifices whether to let Nixops provision SSH deployment keys.
```

```suggestion
        This option specifices whether to let NixOps provision SSH deployment keys.
```

though maybe should describe more about the history, and default behavior, etc.